cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorNonParametrizedTest.java

```diff
@@ -148,4 +148,80 @@ public void testCustomLabelsAndAnnotations(VertxTestContext context) {
                     async.flag();
                 })));
     }
-}
\ No newline at end of file
+
+    public void testCASecretsWithOwnerReference(VertxTestContext context) {
```

Comment:
I think this is missing the `@Test` annotation?

Reply:
True!